exonum-java-binding/core/rust/src/proxy/runtime.rs

```diff
@@ -270,8 +270,20 @@ impl Runtime for JavaRuntimeProxy {
         _context: ExecutionContext,
         _instance_id: InstanceId,
     ) -> Result<(), ExecutionError> {
-        // TODO(ECR-4016): implement
-        Ok(())
+        jni_call_transaction(&self.exec, |env| {
+            // FIXME: as mostly a copy-paste of after_txs, it needs BlockchainData just as well [ECR-4169]
```

Comment:
@vitvakatu BlockchainData native methods did not include these modifications. Java runtime won't work without them.

Reply:
(in afterTransactions)

Reply:
Yep, I'm aware, we need https://github.com/exonum/exonum/pull/1767
(Travis is soooo sloooow)

Reply:
Actually, I will use my fork and implement it right now, in a separate PR